content/recipes/interoperability/c4c-clinical-trials.md

```diff
@@ -0,0 +1,302 @@
+- [BioLINCC](https://biolincc.nhlbi.nih.gov/studies/)
+- [ClinicalStudyDataRequest (CSDR)](https://www.clinicalstudydatarequest.com/)
+- [Health Data Research Innovation Gateway](https://www.healthdatagateway.org/)
+- [NIDDK Central Repository](https://repository.niddk.nih.gov/search/study/)
+- [Project Data Sphere](https://data.projectdatasphere.org/projectdatasphere/html/access)
+- [Immunology Database and Analysis Portal (ImmPort)]()
+- [ITN TrialShare](https://www.immport.org/shared/)
+- [National Sleep Research Resource (NSRR)](https://sleepdata.org/)
+- [National Institute on Drug Abuse NIDA](https://datashare.nida.nih.gov/data)
+- [RDCA-DAP](https://portal.rdca.c-path.org/)
+- [European Genome-phenome Archive (EGA)](https://ega-archive.org/)
+
+The first step created a list of metadata items from the Advanced Search screen on ClinicalTrials.gov. Metadata items from each subsequent repository were compared against this list and mapped across if there was a match. For example, ‘Age Group’ and ‘Age Range’ or ‘Trial Phase’ and ‘Study Phase’. New metadata items that couldn’t be mapped against existing entries were added to the bottom of the list. This gave a good visual of which metadata items occurred most frequently across all of the repositories. The results of the mapping exercise were captured in a Google Sheet. We started with ClinicalTrials.gov because it is the most comprehensive and used repository.
+
+
+
+## Refining the initial metadata list
+The list of metadata items was reviewed by c4c partners, and those not considered cross cutting or common enough for paediatric clinical trials were removed from the list. The original list consisted of 36 items and this was reduced to 27. The following were identified for inclusion:
+
+| Considered cross cutting or common enough to be included in the metadata schema | Not considered cross cutting or common enough to be included in the Metadata Schema|
+| :------------- | :------------- |
+| Study IDs | Rare Disease (tick box yes/no) |
+| Title | Intervention Model |
+| Acronym | NCT Number |
+| Condition or Disease | Consent |
+| Therapeutic Area | Criterion |
+| Indication | Site Name |
+| Study Type | Formulation |
+| Phase | Route of Administration |
+| Funder Type |  |
+| Study Start |  |
+| Sample Size |  |
+| Study Description |  |
+| Status: Recruitment |  |
+| Study Documents |  |
+| Study Results |  |
+| Country |  |
+| Age |  |
+| Age Group |  |
+| Sex |  |
+| Ethnicity |  |
+| Race |  |
+| Additional Inclusion Criteria |  |
+| Additional Exclusion Criteria |  |
+| Outcome Measures |  |
+| Intervention/Treatment |  |
+| IMP with Orphan Designation in the Indication |  |
+| Biospecimens Retained |  |
+| Product Class |  |
+
+## Testing the metadata profile with a representative clinical trial protocol
+
+Each of the above variables were populated (where possible) with information from a clinical trial protocol. They were also mapped to the following ontologies/vocabularies:
```

Comment:
Question/Comment: Why were these vocabularies selected? suggestion = add a statement explaining the reasons for selecting these vocabularies:
-OMOP vocabulary was selected because a number of paediatric trial used OMOP CDM for structuring trial results and study participant information
- NCIT was selected as it is aligned with CDISC vocabulary, and used to markup CDISC SDTM files which are mandated by the regulatory authorities for deposition.
- CTO was selected as it may provide a semantic bridge between CDISC / OMOP representation and the preclinical world where obofoundry resources are extensively used for semantic representation.

 (or something along those lines, this is needed to document and support these choices).